Admin/Extension/PublishableExtension.php

```diff
@@ -41,11 +42,12 @@ public function __construct($formGroup = 'form.group_publish_workflow')
      */
     public function configureFormFields(FormMapper $formMapper)
     {
+        $builder = $formMapper->getFormBuilder()->getFormFactory()->createBuilder(PublishableType::class);
+
         $formMapper
-            ->with($this->formGroup, array('translation_domain' => 'CmfCoreBundle'))
-                ->add('publishable', CheckboxType::class, array(
-                    'required' => false,
-                ))
-            ->end();
+            ->with($this->formGroup, ['translation_domain' => 'CmfCoreBundle'])
+                ->add($builder->get('publishable'))
```

Comment:
We don't have to use `inherit_data` here as we are stealing the `FormType` from the builder. I know stealing fields from a builder is a hack, but it require huge work on the sonata side to allow using a `FormType` natively and still support tabs, groups and fields order.